macaroon-minting.xml

```diff
@@ -85,12 +85,12 @@
     
     <section title="Requesting a macaroon">
       <t>The client requests that the server creates a macaroon by
-      issues an HTTP POST request to the path <spanx
+      issuing an HTTP POST request to the path <spanx
       style="verb">/.well-known/macaroons</spanx>.  The request MUST
       have a <spanx style="verb">Content-Type</spanx> header with
       value <spanx
-      style="verb">application/json+macaroon-request</spanx>.  The
-      request entity must be a valid JSON object.</t>
+      style="verb">application/json</spanx>.  The request entity 
```

Comment:
OK, that would work too.

Another alternative would be to use "/.well-known/macaroons/mint" as the endpoint and simply "application/json" as the Content-Type.  The caveat discovery service would use a different endpoint ("/.well-known/macaroons/caveats" for example).